util/cron/test-perf.cray-cs-hdr.gpu-array-on-device.bash

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+#
+# Run GPU performance tests
+
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common-native-gpu.bash
+export CHPL_COMM=none
+export CHPL_NIGHTLY_TEST_DIRS="gpu/native/array_on_device"
+export CHPL_GPU_MEM_STRATEGY=array_on_device
+
+export CHPL_TEST_PERF_CONFIG_NAME='gpu'
```

Comment:
I don't think this'll work. At least not as expected. `gpu` is the name for our current GPU performance testing config. This will either override tests there, or will crash in some other way. I don't think there's a way to bring two performance configs' results into a single performance page. Pinging @ronawho to see if he can confirm that.

Well, there's `CHPL_TEST_PERF_CONFIGS` and `CHPL_TEST_PERF_DESCRIPTION` that results in checkboxes on the top left [here](https://chapel-lang.org/perf/champs/16-node-cs-hdr/?graphs=all). That seems close enough, but I am not sure if that works correctly if you run tests on completely different set of benchmarks. I guess it won't.

Reply:
I don't think `CHPL_TEST_PERF_CONFIG_NAME` has to be unique (we use chapcs to most chapcs configs). I think `CHPL_NIGHTLY_TEST_CONFIG_NAME` is the important unique one

You're right that `CHPL_TEST_PERF_CONFIGS`/`CHPL_TEST_PERF_DESCRIPTION` is for running the same tests in multiple configurations